ci/test.sh

```diff
@@ -46,37 +48,52 @@ markEndOfBlock "Code Generator Testing"
 markStartOfBlock "Editmode Testing"
 
 pushd "workers/unity"
-  dotnet run -p "${PROJECT_DIR}/tools/RunUnity/RunUnity.csproj" -- \
-    -batchmode \
-    -projectPath "${PROJECT_DIR}/workers/unity" \
-    -runTests \
-    -testPlatform editmode \
-    -logfile "${PROJECT_DIR}/logs/unity-editmode-test-run.log" \
-    -testResults "${EDITMODE_TEST_RESULTS_FILE}"
+    dotnet run -p "${PROJECT_DIR}/tools/RunUnity/RunUnity.csproj" -- \
+        -batchmode \
+        -projectPath "${PROJECT_DIR}/workers/unity" \
+        -runTests \
+        -testPlatform editmode \
+        -logfile "${PROJECT_DIR}/logs/unity-editmode-test-run.log" \
+        -testResults "${EDITMODE_TEST_RESULTS_FILE}"
 
     EDITMODE_TEST_RESULT=$?
 popd
 
 markEndOfBlock "Editmode Testing"
 
-cleanUnity
+cleanUnity "$(pwd)/workers/unity"
```

Comment:
We really need better worker names than `unity`

Reply:
Probably yeah - not sure what we want for this